deployments/overlays/nodeport/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
```

Comment:
question: where is nodeport customize stuff? 
README also empty. is that intentional?

Reply:
we can delete nodeport folder if we don't want this.